src/PythonProjectBootstrapper/ProjectGenerationUtils.py

```diff
@@ -79,31 +82,52 @@ def ConditionallyRemoveUnchangedTemplateFiles(
             original_hash = existing_manifest_dict[removed_file_rel_path]
 
             if current_hash == original_hash:
+                deleted_files.add(removed_full_path.as_posix())
                 removed_full_path.unlink()
 
+    return deleted_files
+
 
 # ----------------------------------------------------------------------
 def CopyToOutputDir(
     src_dir: Path,
     dest_dir: Path,
-) -> None:
+) -> list[set[str]]:
     # Copies all generated files into the output directory and handles the creation/updating of the manifest file
 
     PathEx.EnsureDir(src_dir)
     PathEx.EnsureDir(dest_dir)
 
+    prompt_file = src_dir / "prompt_text.yml"
+    if prompt_file.is_file():
+        shutil.copy2(prompt_file, dest_dir)
```

Comment:
I believe that `shutil.move` can be used instead of `copy2`, eliminating the need for the call to `unlink` on the line below.